pkg/workloadmeta/collectors/internal/containerd/image_sbom_trivy.go

```diff
@@ -44,7 +44,7 @@ func (c *collector) startSBOMCollection(ctx context.Context) error {
 			workloadmeta.EventTypeSet,
 		),
 	)
-
+	ch := make(chan sbom.ScanResult, 2000)
```

Comment:
the channel should be close befor the line 53

Reply:
Thanks for the recommendation. Indeed it looks much better like that.
I applied the changes in the last commit

Reply:
I'm trying to understand all the logic here. But I think it is a mistake to have a channel created here. 

instead we should use the channel present in the `c.sbomScanner` si that the "global scanner" run the scan